x-pack/plugins/ml/server/shared_services/providers/trained_models.ts

```diff
@@ -47,7 +51,7 @@ export interface TrainedModelsProvider {
     putTrainedModel(
       params: estypes.MlPutTrainedModelRequest
     ): Promise<estypes.MlPutTrainedModelResponse>;
-    getELSER(params?: GetElserOptions): Promise<ModelDefinitionResponse>;
+    getELSER(params?: GetModelDownloadConfigOptions): Promise<ModelDefinitionResponse>;
```

Comment:
`getCuratedModelConfig` needs adding to this interface

Reply:
Added in https://github.com/elastic/kibana/pull/172053/commits/43c04fb52e2553c495f1763f6bcc52f2b7dcb959